RiotShareExtension/Sources/ShareItemSender.m

```diff
@@ -641,7 +641,7 @@ - (void)sendVideo:(NSURL *)videoLocalUrl
     {
         if (!RiotSettings.shared.showMediaCompressionPrompt)
         {
-            [MXSDKOptions sharedInstance].videoConversionPresetName = AVCaptureSessionPreset1920x1080;
+            [MXSDKOptions sharedInstance].videoConversionPresetName = AVAssetExportPreset1920x1080;
```

Comment:
This fixes a bug I introduced quite a while ago that I just noticed while testing out this PR.

Reply:
Good catch !